### ISSUE TITLE
Update DRP products from 1.1p to 1.2p in tutorial notebooks

### ISSUE THREAD

**Issue by yymao** [LSSTDESC/DC2-analysis#20]
Update the butler repo in tutorial notebooks to point to 1.2p in tutorial notebooks. 

I think this upgrade can be carried out now, am I right, @wmwv?

(cc @reneehlozek as this is one of the hackuDC2 preparation task)

**Comment by wmwv** [comment]
Yesish.

We need to provide a Jupyter kernel that provides the correct shifter image and `obs_lsstCam` version of the Run 1.2p processing.  @heather999

**Comment by wmwv** [comment]
@yymao @heather999 I confirm that a recent version of the stack: `w_2018_39` can still read the Run1.1p data.

The key difference is that Run 1.1p `obs_lsstSim` camera mapper, while Run 1.2p uses the `obs_lsst[Cam]` mapper.  There's nothing that intentionally broken things for `obs_lsstSim`, but that package is not CIed.

There's no reason this shouldn't have been true, but checking makes me feel much more reassured.

**Comment by heather999** [comment]
We have an initial shifter image that will handle the ongoing Run1.2p reprocessing with w_2018_30+a version of obs_lsstCam.  I'm updating that shifter image to include the python packages that Michael requested.  Once that image is installed at NERSC, then we need to add an additional script that will allow users to add this new image to a jupyter kernel.

**Comment by yymao** [comment]
@heather999 @wmwv thanks. I have two may-be-naive questions:
- Does this mean that we will have a new kernel other than the `desc-stack` that we have been asking people to use? Can we just update this kernel instead of creating a new one?
- (on a different note) what's the path of the 1.2p repo for butler?

**Comment by heather999** [comment]
That's the question.. can we just rename the "desc-stack" kernel? I suppose if the new one is really compatible with Run1.1p, it is ok... but at some point that I expect we will diverge and a new kernel may not be compatible with older data.  I'd like to test to see what happens when we execute "jupyter kernelspec install"and the kernel name already exists in the user's $HOME/.local/share/ area

The Run1.2p data is appearing here (which is a symlink to the projecta area, but I understand that the compute nodes and projecta space + butler do not work well together, so we have link to the scratch area):  /global/cscratch1/sd/desc/DC2/in2p3/Run1.2p/w_2018_30

**Comment by heather999** [comment]
I set up a desc-stack-run1.2p kernel, unfortunately I am unable to log into jupyter-dev, even with MFA set up on my account. While I have opened a ticket, given that it is the weekend, I don't expect to hear from NERSC until tomorrow. There is a new shifter image for `w_2018_30` called `lsstdesc/stack-jupyter:w_2018_30-run1.2p`. I’ve created a new “dev” setup file while we test things out: `/global/common/software/lsst/common/miniconda/kernels/setup-dev.sh` This should set up a new kernel called `desc-stack-run1.2p`.  Perhaps someone else can try running `setup-dev.sh` to see if a new kernel is successfully installed that can be run on jupyter-dev.
